### DISCUSSION TITLE
Comandi generali tapparelle

### DISCUSSION THREAD

**Discussion by cl-7k-01** [indomus/forum#265] [Hardware & Firmware]
Buongiorno a tutti, ho abbandonato l'idea (costosa e complicata) di comandare le tapparelle di casa con comandi generali filari (uno per piano, quindi 2+1 totale).
Ho una raspberry dedicata con HASSIO. Le tapparelle sono tutte comandate da Shelly 2.5 che comando fisicamente da doppi pulsanti interbloccati e "domoticamente" tramite mqtt e HASSIO. Naturalmente in HASSIO ho creato i tre gruppi di tapparelle: piano terra, piano primo, generale, che all'occorrenza comando da app smartphone.
Vorrei però installare in casa (nella classica 503) altri 3 doppi pulsanti interbloccati (per avere la stessa resa estetica) che mi facciano la stessa funzione: uno per comandare la salita e la discesa di tutte le tapparelle del piano terra, uno per tutte le tapparelle del piano primo ecc.
Secondo voi, qual è la via più semplice ed economica da perseguire? Pensavo a 3 ulteriori shelly 2.5 (uno per comando) per rilevare la pressione dei pulsanti, ma come li integro in home assistant?
Grazie in anticipo.

**Comment by bf69** [comment]
ciao, secondo me ti basta uno shelly I3 per ogni gruppo di tapparelle, sono praticamente tre ingressi dove puoi collegare i tuoi interruttori fisici, poi lo integri in HA

**Reply by cl-7k-01** [reply]
Ciao e grazie mille per la risposta!
Ho dato un occhio agli shelly I3, e sono piacevolmente colpito dalle potenzialità e dal costo. Mi resterebbe pure un ingresso per monitorare qualcos'altro.
Con l'I3 dovrei settare una qualcosa tipo:
- interruttore acceso = avvio movimento (es. apertura) fino al completamento del movimento stesso (es. apertura totale);
- interruttore spento = interrompi il movimento.
- dopo un tot di secondi, far tornare l'interruttore nello stato "spento".
Il tutto per il gruppo di tapparelle prescelto ovviamente.
Potrebbe reggere un approccio del genere?

Nella mia testa, da inesperto, immaginavo uno shelly 2.5, collegato ad un doppio pulsante interbloccato, settato nel suo firmware per far muovere un gruppo preciso di tapparelle senza cablarlo fisicamente a nessuna tapparella. Potrebbe reggere?
In questo modo lascerei ad Home Assistant la sola funzione di interfaccia: integro lo shelly via mqtt come qualsiasi altro shelly.
Però il costo sarebbe più del doppio...

**Comment by dante70** [comment]
ciao,
sono d'accordo con chi ti ha suggerito gli shelly I3 ma addirittura secondo me ne bastano 2 in quanto tu avendo 3 doppi pulsanti hai 6 ingressi da monitorare .
Ti do uno spunto giusto se vuoi fare qualcosa "fai date" che e' funzionale ma forse non conveniente in termini di tempo: io ho inserito in alcune 503 un Wemos D1 mini  collegandogli fino a 3 interruttori  agli ingressi in modo da creare 3 entita' su home assistant che poi con le automazioni ho sfruttato per accendere diverse luci 
Ho usato esphome come programmazione e son riuscito anche a "creare", usando un esp32, dei pulsanti touch che esposti su homeassistant mi hanno dato la possibilità di controllare al tocco alcuni faretti

**Reply by cl-7k-01** [reply]
Grazie per il suggerimento e per lo "spunto".
Purtroppo i doppi pulsanti sarebbero dislocati in moda tale da obbligarmi all'acquisto di 3 shelly (in ingresso due doppi pulsanti, uno generale e uno di piano terra e in corridoio al piano primo un doppio pulsante per tutto il piano).
Mi leggerò per bene la guida su InDomus relativa agli shelly i3,
grazie mille
